.github/workflows/msys2.yml

```
@@ -81,6 +82,18 @@ jobs:
         run: |
           python3 -m pip --disable-pip-version-check install gcovr jsonschema pefile pytest pytest-subtests pytest-xdist coverage codecov
 
+      - name: Install pypy3 on x86_64
+        run: |
+          mkdir pypy3local
+          pushd pypy3local
+          wget https://downloads.python.org/pypy/pypy3.8-v7.3.9-win64.zip
+          unzip pypy3.8-v7.3.9-win64.zip
+          mv pypy3.8-v7.3.9-win64/* ./
+          popd 
+          echo $PWD\pypy3local | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+          pypy3 -m ensurepip
+        if ${{ env.MSYS2_ARCH == 'x86_64' }}
```

https://github.com/mesonbuild/meson/actions/runs/2151064770/workflow#L95

```suggestion
        if: ${{ env.MSYS2_ARCH == 'x86_64' }}
```

---

thanks